plat/socionext/synquacer/drivers/scpi/sq_scpi.c

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
In case of SCPI driver, we have a new api to get DRAM info from SCP firmware using SCPI interface: "uint32_t scpi_get_draminfo(struct draminfo *info)". This is part of patch: "synquacer: Retrieve DRAM info from SCP firmware".
I think this could also be accommodated as part of abstraction layer for the shared driver in future.